docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

```diff
@@ -101,3 +101,27 @@ GF_WHITE_LABELING_FOOTER_LINKS_EXTRACUSTOM_URL=http://your.custom.site
 - Grafana version with build number
 
 If you specify `footer_links` or `GF_WHITE_LABELING_FOOTER_LINKS`, then all other default links are removed from the footer and only what is specified is included.
+
+## Custom branding for Public Dashboards
+
+In addition to the customizations described below, as an Enterprise user, you can customize the footer of your public dashboards.
```

Comment:
just fyi that I am removing reference to the user being an Enterprise customer because this the Custom branding topic only applies to Enterprise customers. So it's redundant.

Reply:
Excelent! Thank you!